ykcompile/src/lib.rs

```diff
@@ -510,6 +510,20 @@ impl<TT> TraceCompiler<TT> {
         self.stack_builder.alloc(ty.size(), ty.align())
     }
 
+    fn local_live(&mut self, local: &Local) {
+        match self.variable_location_map.get(local) {
+            Some(Location::NotLive) => {
```

Comment:
Do you need to sync? `Location::NotLive` has been removed.

Reply:
I do.